draft-ietf-quic-transport.md

```diff
@@ -1194,13 +1194,14 @@ server then needs to validate that the client owns the address that it claims.
 QUIC therefore performs source address validation during connection
 establishment.
 
-Servers MUST NOT send more than three times as many bytes as the number of bytes
-received prior to verifying the client's address.  Source addresses can be
+Prior to verifying the client's address, servers MUST NOT send more than three
+times as many bytes as the number of bytes received. Source addresses can be
 verified through an address validation token (delivered via a Retry packet or
-a NEW_TOKEN frame) or by processing any message from the client encrypted using
-the Handshake keys.  This limit exists to mitigate amplification attacks.
+a NEW_TOKEN frame), by processing any message from the client encrypted using
+the Handshake keys, or by processing a TLS ClientHello with a valid
+pre-shared key.  This limit exists to mitigate amplification attacks.
```

Comment:
This last is not sufficient for address validation.

Reply:
That isn't right either.  Willingness to complete the handshake is a decision the server makes, and it does not need to be based on a test of the liveness of the remote address.

It's possible that TLS resumption (0-RTT or not) could bundle a token in the session ticket that allows for address validation, as we used to do.  But we no longer require that to happen.

A server can accept 0-RTT from an unvalidated client address.  The amount that you can send is limited, but it's not unreasonable.  The server doesn't need to send a certificate, so it can probably send some responses.

Reply:
Given this is in the validation section, why not remove this entire sentence?

Reply:
Ah, OK, it's my fundamental misunderstanding.